tests/fixtures/points.py

```
@@ -10,125 +10,272 @@
+    """
+    return find_min_dist(vectors) > threshold
+
+
+def generate_dense_vectors(num: int, size: int, tries=1000) -> list[list[float]]:
+    """
+    Generate a list of dense vectors with a minimum distance check.
+    """
+    vectors = np.random.random(size=(num, size)).round(ROUND_PRECISION).astype(np.float32)
+    tries_counter = 0
+    while not check_distance(vectors):
+        raise RuntimeError(f"Regeneration")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)